docs/concepts/api-extension/custom-resources.md

```diff
@@ -72,6 +223,318 @@ implementations for your custom resources by writing and deploying your own stan
+   </td>
+   <td>Yes
+   </td>
+  </tr>
+  <tr>
+   <td>OpenAPI Schema
+   </td>
+   <td>Is there an OpenAPI (swagger) schema for the types that can be dynamically fetched from the server?  Is the user protected from misspelling field names by ensuring only allowed fields are set?  Are types enforced (e.g. don't put an int in a string field?)
+   </td>
+   <td>No but planned.
+   </td>
+   <td>Yes.
+   </td>
+  </tr>
+</table>
+
+
+
+<table>
+  <tr>
+   <td colspan="2" ><strong>Common Capabilities</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Feature</strong>
+   </td>
+   <td><strong>What it does</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>CRUD
+   </td>
+   <td>The new endpoints support CRUD basic operations via HTTP
+   </td>
+  </tr>
+  <tr>
+   <td>Watch
+   </td>
+   <td>The new endpoints support Kubernetes Watch operations via HTTP
+   </td>
+  </tr>
+  <tr>
+   <td>Discovery
+   </td>
+   <td>Clients like kubectl and dashboard automatically offer list, display, and field edit operations on your resources.
+   </td>
+  </tr>
+  <tr>
+   <td>json-patch
+   </td>
+   <td>The new endpoints support PATCH with Content-Type: <code>application/json-patch+json</code>.
+   </td>
+  </tr>
+  <tr>
+   <td>merge-patch
+   </td>
+   <td>The new endpoints support PATCH with Content-Type: <code>application/merge-patch+json</code>.
+   </td>
+  </tr>
+  <tr>
+   <td>HTTPS
+   </td>
+   <td>The new endpoints uses HTTPS
+   </td>
+  </tr>
+  <tr>
+   <td>Built-in Authentication
+   </td>
+   <td>Can the extension reuse the core apiserver's authentication?
+   </td>
+  </tr>
+  <tr>
+   <td>Built-in 
+<p>
+Authorization
+   </td>
+   <td>Can I reuse the authorization scheme of the Core API server (e.g. RBAC).  
+   </td>
+  </tr>
+  <tr>
+   <td>Finalizers
+   </td>
+   <td>Can I have resource deletion block on another client doing something.
+   </td>
+  </tr>
+  <tr>
+   <td>Initializers
+   </td>
+   <td>Can I have resource creation block on another client doing something.
+   </td>
+  </tr>
+  <tr>
+   <td>UI/CLI Display
+   </td>
+   <td>Kubectl, dashboard can display objects of your new type, without modification.
+   </td>
+  </tr>
+  <tr>
+   <td>Unset vs Empty
+   </td>
+   <td>Can clients distinguish unset fields from set to 0 or empty-string fields.
+   </td>
+  </tr>
+</table>
+
+
+## Preparing to Install a Custom Resource
+
+There are several points to be aware of before adding a custom resource to your cluster.
+
+### Third Party Code and New Points of Failure
+
+Installing a CRD by itself, (meaning creating a resource of kind CustomResourceDefinition), does not cause any "third party code" to run on your apiserver, and does not add any new point of failure.  However packages (such as Charts) or other installation bundles often include CRDs as well as a Deployment of third-party code that implements the business logic for the new Custom Resource.  
+
+Installing an Aggregated APIserver always involves running a new Deployment.  
```

Comment:
I would say, "... always involves running third-party code" to make the connection explicit. Technically you could run it however you want as long as it's accessible through a Service.